examples/other_examples/PyFstat_example_spectrogram.py

```diff
@@ -41,10 +56,11 @@
     "F2": 0,
     "Alpha": 0.0,
     "Delta": 0.5,
-    "tp": data_parameters["tstart"],
-    "asini": 25.0,
-    "period": 50 * 86400,
-    "tref": data_parameters["tstart"],
+    # "tp": data_parameters["tstart"],
+    # "asini": 25.0,
+    # "period": 50 * 86400,
+    # "tref": data_parameters["tstart"],
```

Comment:
These should go back in, just adapting the tstart to timestamps["H1"][0] or something